spec/models/user_spec.rb

```diff
@@ -0,0 +1,5 @@
+require 'rails_helper'
+
+RSpec.describe User, type: :model do
+  pending "add some examples to (or delete) #{__FILE__}"
```

Comment:
Sure thing Matt. Whenever you want to get that started I'll be happy to support that winning choice you're making for our team. Go for it, chief!